sessionctx/variable/tidb_vars.go

```diff
@@ -286,6 +290,7 @@ var (
 	ddlReorgWorkerCounter  int32 = DefTiDBDDLReorgWorkerCount
 	maxDDLReorgWorkerCount int32 = 128
 	ddlReorgBatchSize      int32 = DefTiDBDDLReorgBatchSize
+	ddlErrorCountlimit     int64 = DefTiDBDDLErrorCountLimit
```

Comment:
Why this variable is `int64` and others are `int32`?

Reply:
Because the max value of `ddlErrorCountlimit` is be `MaxInt64` currently.
em... both int64 and int32 are ok for me.

Reply:
Well, At beginning, I think  we should keep consistent with other variables.
Now, I think what you say is reasonable.
This problem is not a big dial.